ts/types/ember-cli/index.d.ts

```diff
@@ -88,5 +95,11 @@ declare module 'ember-cli/lib/models/project' {
     name(): string;
     isEmberCLIAddon(): boolean;
     require(module: string): unknown;
+    isModuleUnification(): boolean;
   }
 }
+
+declare module 'ember-cli-blueprint-test-helpers/helpers';
+declare module 'ember-cli-blueprint-test-helpers/chai';
+declare module 'ember-cli-blueprint-test-helpers/lib/helpers/ember';
+declare module 'mktemp';
```

Comment:
Can we move these declarations out of the ember-cli types? Might make them easier to find if we want to come flesh them out in the future.